src/pages/Landing/index.tsx

```diff
@@ -384,7 +385,12 @@ export default function Landing() {
                 <LearnMoreArrow />
               </LearnMoreContainer>
 
-              <DownloadWalletLink href="https://wallet.uniswap.org/">
+              <DownloadWalletLink
+                {...getDownloadAppLinkProps({
+                  microSiteParams: `utm_source=home_page&utm_medium=webapp&utm_campaign=wallet_microsite&utm_id=1`,
+                  appStoreParams: `ct=Uniswap-Home-Page&mt=8`,
```

Comment:
Make these strings constants still so it's clear what they represent / the case?

Reply:
reopening this one, can we move this object into `src/utils/openDownloadApp.ts` as `landingPageParams`?